components/organisms/o-shipping.vue

```diff
@@ -47,15 +47,12 @@
         :error-message="$t('Field is required')"
         @blur="$v.shipping.streetAddress.$touch()"
       />
+      <!-- https://github.com/building-21/pos-erp/issues/938 -->
       <SfInput
         v-model.trim="shipping.apartmentNumber"
         class="form__element"
         name="apartment-number"
         :label="$t('House/Apartment number')"
-        :required="true"
-        :valid="!$v.shipping.apartmentNumber.$error"
-        :error-message="$t('Field is required')"
-        @blur="$v.shipping.apartmentNumber.$touch()"
```

Comment:
why remove this @blur and the validation ?

Reply:
Aren't we supposed to not want it to be a required field?

By removing those properties we remove the hint 
![image](https://user-images.githubusercontent.com/54149336/117003432-7870ac80-acaa-11eb-9dc1-09a6c96a4b73.png)